### ISSUE TITLE
[$15][MSFT-83] [Member] - The contrast color is low for Unregistered Button

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2794]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges/XXXXXXX
2. Inspect Element
3. Observe the color of Unregistered Button.

**Expected Result**
Expected contrast ratio of  4:5:1.

**Actual Result**
Element has insufficient color contrast Ration 2.5:1 (foreground color: #FFFFFF, background color: ###ff5b52, Contrast Ratio : 3.06:1)

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=ae14bf9a%2D590b%2D4bb4%2Da40a%2D4c6822c880a4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%2083

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096015 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096015 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by r0hit-gupta** [comment]
Please assign me

**Comment by r0hit-gupta** [comment]
PR at https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/8

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096015 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by Dara-K** [comment]
Please change the color of the Unregister button to #EA1900

**Comment by r0hit-gupta** [comment]
@Dara-K I have changed the color. Thanks.

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2794 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2794 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097305 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097305 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097305 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by sushilshinde** [comment]
@veshu @codeMinter @r0hit-gupta  I made a release "topcoder-react-ui-kit": "1.0.4"

Also updated community-app it will be reflected in 20 mins,

Please clear cache and test

**Comment by r0hit-gupta** [comment]
@sushilshinde working for me. Button color is `#ea1900` as Dara-K suggested and passes WCAG AA requirement. 

Test URL - https://test-community-app.topcoder-dev.com/challenges

**Comment by drasticdpk** [comment]
Verified on
Operating System: macOS
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097305<br/><br/>```This is an automated message for crazyk via Topcoder X```
